chain/chain/src/store.rs

```diff
@@ -839,6 +839,38 @@ impl ChainStore {
         }
         Ok(None)
     }
+
+    /// Constructs key 'STATE_SYNC_DUMP:<ShardId>',
+    /// for example 'STATE_SYNC_DUMP:2' for shard_id=2.
+    fn state_sync_dump_progress_key(shard_id: ShardId) -> Vec<u8> {
```

Comment:
we should add a documentation - maybe in https://github.com/near/nearcore/tree/master/docs/misc - (on how this process works (I know that this is still 'experimental' code, but this would help others to understand it better).